oracle/aliases.conf

```
@@ -0,0 +1,23 @@
+###########################
+#
+# Common Oracle Aliases
+#
+###########################
+
+alias cdh='cd $ORACLE_HOME'
+alias cdcrs='cd $ORA_CLU_HOME'
+alias orainv='less $CENTRAL_ORAINV/ContentsXML/inventory.xml'
+
+alias rmansys="rman_"
+alias sqlp="s_"
+alias sqlsys="s_"
+
+
+if [ $CRS_EXISTS -eq 1 ] ; then
```

The variable will be named CBE_CRS_EXISTS, 
```suggestion
if [ $CBE_CRS_EXISTS -eq 1 ] ; then
```